src/directory/directory.mjs

```diff
@@ -189,6 +189,9 @@ export const directory = {
                     },
                     {
                       path: 'src/pages/[platform]/build-a-backend/data/custom-business-logic/connect-amazon-translate/index.mdx'
+                    },
+                    {
+                      path: 'src/pages/[platform]/build-a-backend/data/custom-business-logic/working-with-files/index.mdx'
                     }
```

Comment:
let's move it upwards one level

Reply:
Are you suggesting adding it directly under the data section as its own separate section?